native/android/SwrveSDKPushSupport/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.swrve.unity.gcm">
 
-    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
```

Comment:
This is already done, isn't it?

https://github.com/Swrve/swrve-unity-sdk/blob/release-4_6/native/android/SwrveSDKPushSupport/src/main/AndroidManifest.xml